_posts/2016-09-19-vendors-government-strengthen-partnership-technology-industry-day.md

```diff
@@ -26,7 +26,7 @@ As GSA Administrator Denise Turner Roth said in her opening remarks, “Governme
 
 “For me, the future is a future where every public facing government
 service is easy to understand, is a pleasure to use, works on whatever
-device you’re visit it on, and gives you fast, efficient service,” said
+device you visit it on, and gives you fast, efficient service,” said
```

Comment:
I wasn't sure what this was supposed to be and feel a bit weird "fixing" it because it's a quote... Alternative suggestions welcome!